eZ/Publish/Core/Search/Legacy/Content/Common/Gateway/CriterionHandler/FieldValue/Handler/Keyword.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/**
+ * File containing the DoctrineDatabase Composite field value handler class.
```

Comment:
Description seems to general, maybe use line stating something like `this is a part of eZ Publish Kernel package` - exact wording can be found in codebase.